docs/client/networking.md

```diff
@@ -0,0 +1,210 @@
+# Networking
+
+## Setup
+
+- Transport: QUIC on IPv4, secured by TLS 1.3 with `ED25519` identities
+- Protocol negotiation: [multistream-select 1.0](https://github.com/multiformats/multistream-select/)
+- Multiplexing: Native support by QUIC
+- Gossip: [gossipsub v1](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.0.md)
+
+## Node identification
```

Comment:
It would make sense to specify exactly how the ENR will look since it will deviate from the standard form defined in the [EIP-778](https://eips.ethereum.org/EIPS/eip-778).

Reply:
Added https://github.com/leanEthereum/leanSpec/pull/8/commits/cdcf9ae71d7fea87ecabfe4e03dcf802966cc43a

But I'm not sure how feasible it is to replace the `secp256k1` field with `ed25519` in the ENR implementation, @g11tech @varun-doshi can you have a check? cc: @syjn99 @Kayden-ML

Reply:
All Enr implementations already support this though right. Sigp/enr does why wouldn't the others etc

Reply:
https://github.com/sigp/enr/blob/master/src/lib.rs#L34

Read the comment

Reply:
Yes, ed25519 is more simple for us since the Zig std not support bitcoin secp256k1 right now.

Reply:
@unnawut @KolbyML @g11tech @Vid201 I just read [EIP-778](https://eips.ethereum.org/EIPS/eip-778), it only defines secp256k1 algo and currently only rust ENR implementation extends with ed25519, others such as JS, Go don't have that. And after read [zig-discv5](https://github.com/ChainSafe/zig-discv5) code, its design only support secp256k1 too. So it may be better to stick to secp256k1, I can use this [wrapper](https://github.com/Syndica/secp256k1-zig) to replace Zig std secp256k1 in the [zig-discv5](https://github.com/ChainSafe/zig-discv5).

Reply:
i guess we can go with ed25519 and we can extend 778 EIP into a new one since we are not dealing with beacon clients anyway